app/config/auth.php

```diff
@@ -17,6 +17,13 @@
         'docs' => 'https://appwrite.io/docs/references/cloud/client-web/account#accountCreateMagicURLToken',
         'enabled' => true,
     ],
+    'email' => [
+        'name' => 'Email (OTP)',
+        'key' => 'email',
```

Comment:
```suggestion
        'key' => 'email-otp',
```

Reply:
A bit worried about wotd `token` 😬 In Appwrite (feat SSR) terminology, token is anything that promises to create session in future.
By that logic, email magic URL is also email token. Could we do `emailOtp` instead for hey too?

Reply:
yeap thats fine with me 👍